payments-app-extension-credit-card/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,24 @@
+[[extensions]]
+name = "{{ name }}"
+type = "payments_app_extension"
+handle = "{{ handle }}"
+target = "credit_card"
+
+merchant_label = "Credit Credit Payments App Extension"
+start_payment_session_url = "https://api.paymentprovider.com/endpoint"
+start_refund_session_url = "https://api.paymentprovider.com/refund"
+start_capture_session_url = "https://api.paymentprovider.com/capture"
+start_void_session_url = "https://api.paymentprovider.com/void"
+start_verification_session_url = "https://api.paymentprovider.com/verification"
+supported_countries = ["US"]
+supported_payment_methods = ["visa"]
+supports_3ds = false
+supports_installments = false
+supports_deferred_payments = false
+test_mode_available = true
+api_version = "2021-07"
+encryption_certificate = {}
+multiple_capture = false
+confirmation_callback_url = "https://api.paymentprovider.com/confirm"
+ui_extension_registration_uuid = "6f5af7b2-0946-4873-a8a8-089f66414297"
```

Comment:
What's this registration uuid? The UI extension?

Reply:
it maps to the registration uuid

Reply:
it shouldn't have a default value then no?

Reply:
i think i'll remove it for now. we have a ticket to deal with this: https://github.com/Shopify/payments-platform/issues/4383